src/featureforest/_sam_rf_segmentation_widget.py

```diff
@@ -20,7 +20,7 @@
 import numpy as np
```

Comment:
This file should probably be renamed!

Reply:
Yes, I left this for another PR, or maybe a direct update to the main branch.